docs/source/commandline/commandline-all.rst

```diff
@@ -1592,32 +2213,45 @@ Core Communication Options
 
 **-comm_localhost** *<string>*
 
-	Local address to bind to. This can be a traditional ``xxx.xxx.xxx.xxx`` address or a string containing a resolvable hostname. The default is value is "``0.0.0.0``"
+    Local address to bind to. This can be a traditional ``xxx.xxx.xxx.xxx``
+    address or a string containing a resolvable hostname.
+
+    The default is value is "``0.0.0.0``"
 
 .. _mame-commandline-commlocalport:
 
 **-comm_localport** *<string>*
 
-	Local port to bind to. This can be any traditional communications port as an unsigned 16-bit integer (0-65535). The default value is "``15122``".
+    Local port to bind to. This can be any traditional communications port as
+    an unsigned 16-bit integer (0-65535).
+
+    The default value is "``15122``".
 
 .. _mame-commandline-commremotehost:
 
 **-comm_remotehost** *<string>*
 
-	Remote address to connect to. This can be a traditional xxx.xxx.xxx.xxx address or a string containing a resolvable hostname. The default is value is "``0.0.0.0``"
+    Remote address to connect to. This can be a traditional xxx.xxx.xxx.xxx
+    address or a string containing a resolvable hostname.
+
+    The default is value is "``0.0.0.0``"
```

Comment:
Same as the one above - could do with an explanatory note.

Reply:
Done.